doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1577,62 +1615,188 @@ Response code:
          }
         ]
 
-### updateContext [POST /v2/updateContext]
+### Register [POST /v2/op/register]
 
+This operation allows to create, update and/or delete several registrations in a single batch operation.
 The payload is an object with two properties:
 
-* `action`, to specify the kind of updateContext action to do (see OMA NGSI specification for the semantics),
-   either UPDATE, APPEND or DELETE
-* `entities`, an array of entities, each one specified using the the JSON entity representation format
-  (described in a section above).
+* `actionType`, to specify the kind of register action to do: either APPEND, UPDATE, DELETE.
+* `registrations`, an array of registration, each one specified using the JSON registration representation
+  format (described in a section above). In the case of APPEND operation, the registration `id` must not be
+  included.
 
 Response:
 
-* Successful operation uses 201 Created.
+* Successful UPDATE or DELETE operation uses 204 No Content.
+* Successful APPEND operation uses 201 Created and returns a list of IDs, each one corresponding to the ID of the
+  element in the request payload in the same order.
```

Comment:
From http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html, Location cannot be multivaluated. So, we will use 200 OK with a payload (as in the original proposal, but using 200 instead of 201).